pkg/virt-api/rest/subresource_test.go

```diff
@@ -297,6 +300,86 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 			close(done)
 		})
 
+		It("should ForceRestart VirtualMachine", func(done Done) {
```

Comment:
I think we need a test which tests the scenario where no pod is present for the VMI. Can you create a VM, where you attach a non-existing PVC? If the PVC does not exist, we can't create the pod. Then force-delete it and check if everything works without issues.

Reply:
Added a Unit test where there are no pods and `findPod` func returns an error and we expect that error; For functional test with this scenario if VM is not running it's already covered as part of this https://github.com/kubevirt/kubevirt/blob/master/tests/subresource_api_test.go#L104